src/lib/plugin_apis/fs.api

```diff
@@ -929,6 +992,34 @@ guint64 bd_fs_get_size (const gchar *device, GError **error);
  */
 guint64 bd_fs_get_free_space (const gchar *device, GError **error);
 
+/**
+ * BDFSMkfsOptionsFlags:
+ * Flags indicating mkfs options are available for given filesystem type.
+ */
+typedef enum {
+    BD_FS_MKFS_LABEL     = 1 << 0,
+    BD_FS_MKFS_UUID      = 1 << 1,
+    BD_FS_MKFS_DRY_RUN   = 1 << 2,
+    BD_FS_MKFS_NODISCARD = 1 << 3,
+} BDFSMkfsOptionsFlags;
+
+/**
+ * bd_fs_can_mkfs:
+ * @type: the filesystem type to be tested for installed mkfs support
+ * @options: (out): flags for allowed mkfs options (i.e. support for setting label or UUID when creating the filesystem)
+ * @required_utility: (out) (transfer full): the utility binary which is required for creating (if missing i.e. returns FALSE but no error)
```

Comment:
*if missing i.e. returns FALSE* ? Extra *i.e.*, I suppose.

Reply:
Changed, hope it makes more sense now.